### PR TITLE
Create 2019-08-06-OPA-Gatekeeper-Policy-and-Governance-for-Kubernetes.md

### PR DESCRIPTION
**Pull request by yixin21**
zh-trans 2019-08-06-OPA-Gatekeeper-Policy-and-Governance-for-Kubernetes.md

ref:k8smeetup/website-tasks#3356

### CODE REVIEW

**Comment by yixin21** [comment]
Thanks for your review
/assign @sataqiu

File: content/zh/blog/_posts/2019-08-06-OPA-Gatekeeper-Policy-and-Governance-for-Kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+If your organization has been operating Kubernetes, you probably have been looking for ways to control what end-users can do on the cluster and ways to ensure that clusters are in compliance with company policies. These policies may be there to meet governance and legal requirements or to enforce best practices and organizational conventions. With Kubernetes, how do you ensure compliance without sacrificing development agility and operational independence? 
+--->
+## 出发点 
+
+如果您所在的组织一直在使用 Kubernetes，您可能一直在寻找如何控制终端用户在集群上的行为，以及如何确保集群符合公司政策。这些策略可能需要满足管理和法律要求，或者符合最佳执行方法和组织惯例。使用 Kubernetes，如何在不牺牲开发敏捷性和运营独立性的前提下确保合规性？
+
+<!--
+For example, you can enforce policies like:
+
+* All images must be from approved repositories
```
Comment (markthink) [inline content/zh/blog/_posts/2019-08-06-OPA-Gatekeeper-Policy-and-Governance-for-Kubernetes.md:49]: image 应该翻译为镜像，不应该是图像..

Comment (yixin21) [reply content/zh/blog/_posts/2019-08-06-OPA-Gatekeeper-Policy-and-Governance-for-Kubernetes.md:49]: 好的，已修改，非常感谢

**Comment by yixin21** [comment]
> 
> 
> Thanks @yixin21
> Please squash commits into one, then we can merge this PR.
> /approve

Could you please help to squash commits into one? It's hard to operate for my environment configuration.
Thanks @SataQiu

**Pull request merged**